cmd/ike/cmd/develop.go

```diff
@@ -107,6 +109,7 @@ func NewDevelopCmd() *cobra.Command {
 	developCmd.Flags().Bool("no-build", false, "always skips build")
 	developCmd.Flags().StringSliceP("watch", "w", []string{currentDir()}, "list of directories to watch")
 	developCmd.Flags().StringSlice("watch-exclude", excludeTpLog, "list of patterns to exclude (defaults to telepresence.log which is always excluded)")
+	developCmd.Flags().Int64("watch-interval", 500, "watch interval (in ms)")
```

Comment:
Don't think I would have exposed this. Does a user really care?

Reply:
Right, maybe it's only useful for tests.

Reply:
Made it hidden 7626fe9